open-metadata-implementation/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/configuration/registration/GovernanceServicesDescription.java

```diff
@@ -88,15 +90,33 @@
         return serviceDescriptionList;
     }
 
+    /**
+     * Return a set of non null url markers (short names) of the governance services
```

Comment:
Using a set means the order will vary.  Also need to be clear - are these these governance services or the governance servers - fo the UI I think they are the servers?